### PR TITLE
[DotNetCoreCLI] Using different library for searching files

### PR DESCRIPTION
**Pull request by desattir**

### CODE REVIEW

File: Tasks/DotNetCoreCLI/Tests/mock-findfiles.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export function findFiles (projects: string, includeFolder: boolean) : string[] {
+export function resolveFilterSpec (projects: string, includeFolder: boolean) : string[] {
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/Tests/mock-findfiles.ts]: includeFolder should also  be changed.

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 2]
```diff
@@ -72,7 +72,8 @@ export class dotNetExe {
             }
             else {
                 var pattern = path.dirname(projectFile) + "/**/publish";
-                var files = ffl.findFiles(pattern, true);
+                var files = nutil.resolveFilterSpec(pattern,
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: Make sure that this is working with e2e tests.

Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: This path is exercised when the output argument is missing.

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ export class dotNetExe {
             }
             else {
                 var pattern = path.dirname(projectFile) + "/**/publish";
-                var files = ffl.findFiles(pattern, true);
+                var files = nutil.resolveFilterSpec(pattern, "", true, true);
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: is the empty folder (2nd parameter) expected here?

Comment (desattir) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: Giving it as empty bypasses the baseFolder check. And it has to be a string, I can either give a "" or null then.

**Pull request merged**